.travis.yml

```diff
@@ -119,11 +121,6 @@ jobs:
     - php: 7.2
       env: WP_MULTISITE=1
       if: NOT commit_message =~ js_only AND NOT commit_message =~ dependabot AND NOT sender =~ dependabot
-    # wp 4.5 builds
-    - php: 5.5
-      dist: trusty
-      env: WP_VERSION=4.5
-      if: NOT commit_message =~ js_only AND NOT commit_message =~ dependabot AND NOT sender =~ dependabot
```

Comment:
shaved like 12-15 minutes off of the tests times by removing this :)

Reply:
cool 👍